bofire/surrogates/single_task_gp.py

```diff
@@ -52,7 +54,7 @@ def get_scaler(
             key
             for key, value in input_preprocessing_specs.items()
             if value != CategoricalEncodingEnum.DESCRIPTOR
-            and not isinstance(value, MolFeatures)
+            and not isinstance(value, MordredDescriptors)
```

Comment:
@simonsung06: At some point we really have to tidy this up.

Reply:
I have made a small changes so that it uses the `get_continuous_features` and `get_feature_indices` functions.

Reply:
This is convenient small change for now. We can definitely talk about changing this further/moving it somewhere else potentially ;)